<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>main_desktop</class>
 <widget class="QWidget" name="main_desktop">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1024</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>main_desktop</string>
  </property>
  <widget class="QPushButton" name="compassButton">
   <property name="geometry">
    <rect>
     <x>60</x>
     <y>500</y>
     <width>80</width>
     <height>80</height>
    </rect>
   </property>
   <property name="text">
    <string>正北</string>
   </property>
  </widget>
  <widget class="QPushButton" name="cameraButton">
   <property name="geometry">
    <rect>
     <x>220</x>
     <y>500</y>
     <width>81</width>
     <height>81</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-image:url(:/camera.png)</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QPushButton" name="lockButton">
   <property name="geometry">
    <rect>
     <x>320</x>
     <y>500</y>
     <width>81</width>
     <height>81</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-image:url(:/lock.png)</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QPushButton" name="recordButton">
   <property name="geometry">
    <rect>
     <x>430</x>
     <y>500</y>
     <width>81</width>
     <height>81</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-image:url(:/record.png)</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QPushButton" name="setFirstButton">
   <property name="geometry">
    <rect>
     <x>530</x>
     <y>500</y>
     <width>81</width>
     <height>81</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-image:url(:/setting.png)</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QPushButton" name="movieButton">
   <property name="geometry">
    <rect>
     <x>630</x>
     <y>500</y>
     <width>81</width>
     <height>81</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-image:url(:/movie1.png)</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QPushButton" name="camera_change_Button">
   <property name="geometry">
    <rect>
     <x>840</x>
     <y>500</y>
     <width>81</width>
     <height>81</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-image:url(:/change.png)</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_8">
   <property name="geometry">
    <rect>
     <x>880</x>
     <y>10</y>
     <width>50</width>
     <height>50</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="time_Label">
   <property name="geometry">
    <rect>
     <x>629</x>
     <y>10</y>
     <width>241</width>
     <height>51</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">color:rgb(244, 255, 29);
font: 75 14pt &quot;Courier 10 Pitch&quot;;</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="cameraView">
   <property name="geometry">
    <rect>
     <x>-1</x>
     <y>-1</y>
     <width>1031</width>
     <height>601</height>
    </rect>
   </property>
   <property name="sizePolicy">
    <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
     <horstretch>0</horstretch>
     <verstretch>0</verstretch>
    </sizepolicy>
   </property>
   <property name="styleSheet">
    <string notr="true">background-image:url(:/picture.png)</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <zorder>cameraView</zorder>
  <zorder>compassButton</zorder>
  <zorder>cameraButton</zorder>
  <zorder>lockButton</zorder>
  <zorder>recordButton</zorder>
  <zorder>setFirstButton</zorder>
  <zorder>movieButton</zorder>
  <zorder>camera_change_Button</zorder>
  <zorder>pushButton_8</zorder>
  <zorder>time_Label</zorder>
  <zorder>horizontalSpacer</zorder>
  <zorder>horizontalSpacer_2</zorder>
  <zorder>horizontalSpacer_3</zorder>
  <zorder>horizontalSpacer_4</zorder>
  <zorder>horizontalSpacer_5</zorder>
  <zorder>horizontalSpacer_6</zorder>
  <zorder>horizontalSpacer_7</zorder>
  <zorder>horizontalSpacer_8</zorder>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
